python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -606,40 +624,55 @@ def helptext(f, s):
 
     def _init(self, *args, **kwds):
 
-        # The first thing we do is, if we were called with GPIProxyClassFactory(_existing=some_ganga_object)
-        # to simply create a link between them and us and nothing else.
-        if '_existing' in kwds:
-            # Link the two together in the proxy layer
-            setattr(self, implRef, kwds['_existing'])
-            setattr(kwds['_existing'], proxyObject, self)
-            return
+        ## Zero-th fully initialize self before moving on
+        super(type(self), self).__init__()
```

Comment:
I wonder if changing it to:
```python
GPIProxyObject.__init__(self)
```
would be more explicit and safer. Otherwise I agree that we never subclass these proxy classes so there's probably no danger.